sopel/plugin.py

```diff
@@ -928,12 +932,19 @@ def handle_example_bugs(bot, trigger, match):
 
         More than one pattern can be provided as positional argument at once.
 
+    .. versionchanged:: 7.1
+
+        The ``trigger`` parameter now represents the same match as the
+        ``match`` parameter, making ``match`` an obsolete parameter, kept
+        for backward compatibility.
+
     .. seealso::
 
         To detect URLs, Sopel uses a matching pattern built from a list of URL
         schemes, configured by
         :attr:`~sopel.config.core_section.CoreSection.auto_url_schemes`.
 
+    .. __: https://docs.python.org/3/library/re.html#match-objects
```

Comment:
Just as a side note, it's possible to link this via intersphinx through `:ref:` instead of hard-coding the URL here.

Reply:
Oh? I couldn't find it! I guess I was too tired when I wrote that. (I worked after midnight a couple of time on this so... that might be why)